matador/utils/chem_utils.py

```diff
@@ -643,3 +645,36 @@ def get_formula_from_stoich(stoich, elements=None, tex=False, sort=True, latex_s
                 else:
                     form += elem[0] + str(int(elem[1]))
     return form
+
+def magres_reference_shifts(
+    magres: list,
+    reference: dict,
+    shielding_key: str = "chemical_shielding_iso",
+    shift_key: str = "chemical_shift_iso",
+):
+    """ Convert chemical shielding to chemical shift.
+
+    Parameters:
+        magres (list): list of containing magres data.
+        reference (str): dictionary of conversion values in the form {element: [gradient, constant]}.
```

Comment:
```suggestion
        reference (Dict[str, Tuple[float, float]]): dictionary of conversion values in the form {element: [gradient, constant]}.
```

will also need to do `from typing import Dict, Tuple` (and `Any` for the other comment) at the top of the module.